src/main/java/com/cburch/logisim/fpga/hdlgenerator/FileWriter.java

```diff
@@ -152,42 +153,24 @@ public static File GetFilePointer(String TargetDirectory, String Name) {
     ArrayList<String> Lines = new ArrayList<>();
     if (HDL.isVHDL()) {
       Lines.add("--==============================================================================");
-      Lines.add("--== Logisim goes FPGA automatic generated VHDL code                          ==");
+      Lines.add("--== Logisim-evolution goes FPGA automatic generated VHDL code                ==");
+      Lines.add(String.format("--== %-72s ==", Main.APP_URL));
       Lines.add("--==                                                                          ==");
```

Comment:
Please replace this long white space portion by [`<string>.repeat()`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#repeat(int)) (multiple places)

Reply:
Is it considered best practice to do it this way?
Or why do you propose to change it?
I personally prefer how it currently is, because it is much clearer and easier to understand.

Reply:
It makes code more compact and cannot. Also it is instantly clear how many spaces, `*` is there etc. If done right, there'd be no need for `magic constant` like  `33` here:

```java
    ....repeat(Math.max(0, 33 - Main.APP_NAME.length()))
```

but you'd have i.e. `var line_width = 33;` and all the other lines widths related to that. If at one day you like to change the width, change of that var would be the only thing needed.